zuul.d/jobs.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
@son-vyas Since we merged the base job here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/tcib.yaml#L3
lets use it as a base and add the tcib job.
```
- job:
    name: tcib-build-containers
    parent: cifmw-tcib-base
    irrelevant-files:
      - .*/*.md
      - ^.github/.*$
      - ^LICENSE$
      - ^OWNERS$
      - ^OWNERS_ALIASES$
      - ^PROJECT$
      - ^README.md$
```